via/static/styles/video_player.css

```diff
@@ -3,15 +3,26 @@ CSS entry point for the video player app.
 */
 
 @tailwind base;
-
-body {
-  @apply bg-grey-0;
-  @apply font-sans text-sm text-color-text;
-}
-
 @tailwind components;
 @tailwind utilities;
 
+@layer base {
+  body {
+    @apply bg-grey-0;
+    @apply font-sans text-sm text-color-text;
+  }
+
+  /*
+  Input fields must have a minimum size on iOS to avoid the browser zooming in
+  when they are focused.
+  */
+  @media(pointer: coarse) {
+    input, textarea {
+      font-size: max(16px, 100%);
+    }
+  }
```

Comment:
Should we upstream this into the frontend-shared package?

Reply:
Actually, I want to think about that. I've found so far that it's easier to wrangle base styles per-app, as that's where our variation tends to be. Not sure. Perhaps we could track in a separate ticket?

Reply:
> Perhaps we could track in a separate ticket?

Will do 👍

Reply:
https://github.com/hypothesis/frontend-shared/issues/1159